cpp/daal/include/algorithms/decision_forest/decision_forest_regression_training_types.h

```diff
@@ -72,7 +72,8 @@ enum InputId
 {
     data              = algorithms::regression::training::data,               /*!< %Input data table */
     dependentVariable = algorithms::regression::training::dependentVariables, /*!< %Values of the dependent variable for the input data */
-    lastInputId       = dependentVariable
+    weights           = algorithms::regression::training::weights,            /*!< %Optional. Weights of the observations in the training data set */
```

Comment:
Please add weights support for oneAPI interfaces also and its transfer into CPU backend.

Reply:
After offline conversation with Mikhail, it was decided not to add weights to GPU interfaces yet